src/__tests__/patients/patient-slice.test.ts

```diff
@@ -203,6 +203,11 @@ describe('patients slice', () => {
       const expectedPatient = {
         id: expectedPatientId,
         givenName: undefined,
+        dateOfBirth: addDays(new Date(), 4).toISOString(),
+        suffix: '061002',
+        prefix: '061002',
+        familyName: '061002',
+        preferredLanguage: '061002',
```

Comment:
This test was changed to only check for required fields. I think that we should remove everything but `givenName` in this test. 

I think we'll need to add some other test which checks for numbers in the string.

Reply:
I believe this is why the build is failing.